src/test/java/api/loans/scenarios/RefundLostItemFeesBaseTest.java

```diff
@@ -0,0 +1,470 @@
+package api.loans.scenarios;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.matchers.AccountActionsMatchers.CANCELLED_ITEM_RETURNED;
+import static api.support.matchers.AccountActionsMatchers.CREDITED_FULLY;
+import static api.support.matchers.AccountActionsMatchers.REFUNDED_FULLY;
+import static api.support.matchers.AccountActionsMatchers.REFUND_TO_BURSAR;
+import static api.support.matchers.AccountActionsMatchers.REFUND_TO_PATRON;
+import static api.support.matchers.AccountActionsMatchers.arePaymentRefundActionsCreated;
+import static api.support.matchers.AccountActionsMatchers.areTransferRefundActionsCreated;
+import static api.support.matchers.AccountActionsMatchers.isCancelledItemReturnedActionCreated;
+import static api.support.matchers.AccountMatchers.isClosedCancelledItemReturned;
+import static api.support.matchers.AccountMatchers.isOpen;
+import static api.support.matchers.AccountMatchers.isPaidFully;
+import static api.support.matchers.AccountMatchers.isRefundedFully;
+import static api.support.matchers.AccountMatchers.isTransferredFully;
+import static api.support.matchers.ItemMatchers.isLostAndPaid;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanAccountActionsMatcher.hasLostItemFeeActions;
+import static api.support.matchers.LoanAccountActionsMatcher.hasLostItemProcessingFeeActions;
+import static api.support.matchers.LoanAccountMatcher.hasLostItemFee;
+import static api.support.matchers.LoanAccountMatcher.hasLostItemProcessingFee;
+import static api.support.matchers.LoanAccountMatcher.hasNoOverdueFine;
+import static api.support.matchers.LoanAccountMatcher.hasOverdueFine;
+import static api.support.matchers.LoanMatchers.isClosed;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInRelativeOrder;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTime.parse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+public abstract class RefundLostItemFeesBaseTest extends APITests {
```

Comment:
Personally I think `test base` reads better (or possibly without the base at all).

```suggestion
public abstract class RefundLostItemFeesTestBase extends APITests {
```

```suggestion
public abstract class RefundLostItemFeesTests extends APITests {
```

Reply:
I'll proceed with `RefundLostItemFeesTestBase`.